### ISSUE TITLE
creating tools to easaly edit the manifest

### ISSUE THREAD

**Issue by EIA485** [neos-modding-group/neos-mod-manifest#75]

**Comment by XDelta** [comment]
Yeah, it'd be pretty neat to have a part of the website that you can just paste your mod link in and have several fields filled out for you. At least providing most of the fields for your single manifest entry.

**Comment by rampa3** [comment]
> Yeah, it'd be pretty neat to have a part of the website that you can just paste your mod link in and have several fields filled out for you. At least providing most of the fields for your single manifest entry.

Would be neat. Maybe the Neos OAuth as primitive as it is now would be useful here? Theoretically, I think having dev accounts with a mod release control panel gated behind a login would be a way to go here from my POV. And when such infrastructure would be implemented, we could technically leave git as the backbone, since everything user/dev/reviewer facing would be done on the website.

**Comment by ljoonal** [comment]
> Maybe the Neos OAuth as primitive as it is now would be useful here?

I'd not mix Neos and Github accounts, just using Github's own Oauth2 should be fine.

**Comment by rampa3** [comment]
> > Maybe the Neos OAuth as primitive as it is now would be useful here?
> 
> I'd not mix Neos and Github accounts, just using Github's own Oauth2 should be fine.

But what if you develop GitHub less - everyone working on Neos stuff will have Neos account. And when I talked about control panel, the idea was to hide all the git stuff in the page (Having the page's program doing the pushes when authorized in page by reviewer.) and have user interact with the page, and that is it.

**Comment by XDelta** [comment]
Using githubs Oauth system would work in that most people who've made mods have the source and releases hosted on github already. For Using Neos's Oauth stuff, it is less documented and going to be less featured. I say go with using Github's
